subprojects/docs/src/docs/userguide/migration/migrating_from_maven.adoc

```diff
@@ -37,7 +37,7 @@ In-depth performance comparison and business cases for switching from Maven to G
 == General guidelines
 
 Gradle and Maven have fundamentally different views on how to build a project.
-Gradle provides a flexible and extensible build model that delegates the actual work to a <<what_is_gradle#the_core_model_is_based_on_tasks,_graph of task dependencies_>>.
+Gradle provides a flexible and extensible build model that delegates the actual work to a graph of task dependencies.
```

Comment:
I wonder if "task dependencies" is clear enough here?  Would a new user coming from Maven understand this?

Reply:
```suggestion
Gradle provides a flexible and extensible build model that delegates the actual work to the execution of a graph of tasks.
```